cmd/tmsp-cli/tmsp-cli.go

```diff
@@ -15,10 +16,47 @@ import (
 	"github.com/urfave/cli"
 )
 
+//structure for data passed to print response
+// variables must be exposed for JSON to read
+type pr struct {
+	Res       types.Result
+	S         string
```

Comment:
was just about to comment on this, This is a relic from the old code. Holds information typically a variant of `Res.Data` in a string (but sometimes slightly different). I've just changed `S` to `Data`